### ISSUE TITLE
Alt key handling is mostly broken

### ISSUE THREAD

**Issue by medzernik** [WootingKb/wooting-macros#123]

**Comment by chrishenn** [comment]
ooh this is very sad. I'd love to have a cross-platform, modern hotkey app that supports all the features of those that came before. 

Glancing at the source for autohotkey, which appears to do a careful dance to handle win- and alt- modifier shortcuts, I can see why it would be a headache - just a wall of quirks and oddities mentioned here
https://github.com/AutoHotkey/AutoHotkey/blob/ea437ce78c426c5f0a0644f68bb675cf59acdc36/source/keyboard_mouse.cpp#L1280

**Comment by medzernik** [comment]
> ooh this is very sad. I'd love to have a cross-platform, modern hotkey app that supports all the features of those that came before.
> 
> Glancing at the source for autohotkey, which appears to do a careful dance to handle win- and alt- modifier shortcuts, I can see why it would be a headache - just a wall of quirks and oddities mentioned here https://github.com/AutoHotkey/AutoHotkey/blob/ea437ce78c426c5f0a0644f68bb675cf59acdc36/source/keyboard_mouse.cpp#L1280

Indeed. In 1.2 or 1.1.1, I think I'll disable the ALT key as a trigger entirely (on Windows). It's too much of a headache until it's fixed

**Comment by Martmists-GH** [comment]
It seems alt handling on Linux is broken as well, I can configure it as left alt, but then detection only sees it as alt? And right alt is detected as `Unknown(0)`
